uniffi_macros/src/lib.rs

```diff
@@ -22,12 +22,15 @@ use syn::{punctuated::Punctuated, LitStr, Token};
 /// environment to let it load the component bindings, and will pass iff the script
 /// exits successfully.
 ///
-/// To use it, invoke the macro with one or more file paths relative to the crate root
-/// directory. It will produce one `#[test]` function per file, in a manner designed to
+/// To use it, invoke the macro with the idl file as the first argument, then
+/// one or more file paths relative to the crate root directory.
+/// It will produce one `#[test]` function per file, in a manner designed to
 /// play nicely with `cargo test` and its test filtering options.
 #[proc_macro]
 pub fn build_foreign_language_testcases(paths: proc_macro::TokenStream) -> proc_macro::TokenStream {
-    let paths = syn::parse_macro_input!(paths as FilePaths).0;
+    let mut paths = syn::parse_macro_input!(paths as FilePaths).0;
```

Comment:
I think we should refactor this to be called like `build_foreign_language_testcases("file.idl", ["array", "of", "test", "files"])`, and I think it would be a great opportunity for someone who hasn't done it before, to try messing around with `syn`/`quote` and the infra of doing proc macros. It can be a follow-up though.